release-tool

```diff
@@ -878,8 +895,17 @@ appsign() {
 
                 # overwrite the original exe with the signed exe
                 mv -f "${f}.signed" "${f}"
+            elif [[ ${f: -4} == ".msi" ]]; then
+                # osslsigncode does not succeed at signing MSI files at this time...
+                logInfo "Signing file '${f}' using Microsoft signtool..."
+                signtool sign -f "${signtool_key}" -p "${password}" -d "KeePassXC" \
```

Comment:
You need to re-add the signtool check if we need to use this here.

Reply:
Corrected in last push